src/scenes/home/index.jsx

```diff
@@ -98,6 +106,27 @@ export const Home = () => {
     }
 
   }
+  
+  const [modalIsOpen, setIsOpen] = React.useState(false);
+
+  function openModal(id) {
+    setIsOpen(true);
+    api.get(`/cards/${id}`)
```

Comment:
We already have this data in state on the main component, so I don't think we need to fire off another API request to get the same data. And in fact, we don't really need separate states for modal description and title: we can just use the state we already have on `cards`.